model/manifest.go

```diff
@@ -330,6 +333,10 @@ func (m *Manifest) IsValid() error {
 		return errors.New("invalid SupportURL")
 	}
 
+	if m.SupportURL != "" && !IsValidHttpUrl(m.ReleaseNotesURL) {
```

Comment:
The logic of the other checks is wrong. The community member who submitted the code, will fix it. See https://github.com/mattermost/mattermost-server/issues/13761

Reply:
Shouldn't the check here for empty be for `ReleaseNotesURL`?
```suggestion
	if m.ReleaseNotesURL != "" && !IsValidHttpUrl(m.ReleaseNotesURL) {
```